runtime/commonMain/src/kotlinx/benchmark/internal/KotlinxBenchmarkPluginInternalApi.kt

```diff
@@ -0,0 +1,19 @@
+package kotlinx.benchmark.internal
+
+/**
+ * Marks declarations that are **internal** to the Kotlinx Benchmark project.
+ *
+ * Classes or functions marked with this annotation have `public` visibility
+ * for historical or technical reasons. They are not intended for use by
+ * Kotlinx Benchmark users. They have no behaviour guarantees, and may lack clear
+ * semantics, documentation, and backward compatibility.
+ *
+ * Please remove references to internal API to supported stable references.
+ * If you still require access to these APIs, please raise an issue.
+ */
+@RequiresOptIn(
+    message = "This API is internal to the Kotlinx Benchmark project and will be hidden in a future release.",
```

Comment:
This annotation might not necessarily be used on declarations that will be made internal. I suggest changing the message to match [the other opt-in annotation](https://github.com/Kotlin/kotlinx-benchmark/blob/b8e07eb4d70b5f39654c8178d5dd0f8b92fc70c9/plugin/main/src/kotlinx/benchmark/gradle/internal/KotlinxBenchmarkPluginInternalApi.kt#L17). 

```suggestion
    message = "Internal Kotlinx Benchmark API. It may be changed in future releases without notice.",
```